core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data-editor-panel.service.spec.ts

```diff
@@ -41,8 +39,6 @@ describe('Training Data Editor Panel Service', function() {
       'TrainingDataEditorPanelService');
     $uibModal = $injector.get('$uibModal');
     AlertsService = $injector.get('AlertsService');
-    ExternalSaveService = $injector.get('ExternalSaveService');
```

Comment:
Maybe these lines can be removed too:

https://github.com/oppia/oppia/blob/b1ba8d11e81b968cddaaa66180f8b787e7c72372/core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data-editor-panel.service.spec.ts#L30-L35

Reply:
Done.